src/common/transformations/tests/CMakeLists.txt

```diff
@@ -29,18 +29,28 @@ ov_add_test_target(
     ROOT ${CMAKE_CURRENT_SOURCE_DIR}
     DEPENDENCIES
     LINK_LIBRARIES
-        gtest
-        gtest_main
-        openvino::runtime::dev
-        commonTestUtils
-        lptNgraphFunctions
         gmock
         funcTestUtils
-        inference_engine_legacy
         offline_transformations
         sharedTestClasses
+        lptNgraphFunctions
```

Comment:
Do you need lptNgraphFunctions?

Reply:
yes, build fails without this dependency